frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -32,6 +33,7 @@ interface IReviewProps {
 
 export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
   const formikProps = useFormikContext();
+  const netBookYear = getIn(formikProps.values, 'data.financials.0.netbook.fiscalYear');
```

Comment:
Let's try and use withNamespace as much as possible rather then hardcoding in the object paths.

Reply:
Fixed!